kafka/src/main/java/com/expedia/adaptivealerting/kafka/visualizer/ElasticSearchService.java

```diff
@@ -0,0 +1,48 @@
+package com.expedia.adaptivealerting.kafka.visualizer;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.http.HttpHost;
+import org.apache.http.entity.ContentType;
+import org.apache.http.nio.entity.NStringEntity;
+import org.elasticsearch.client.Request;
+import org.elasticsearch.client.Response;
+import org.elasticsearch.client.RestClient;
+import org.elasticsearch.client.RestClientBuilder;
+
+
+@Slf4j
+public class ElasticSearchService implements Runnable {
+
+    private static String HOST = "localhost";
+    private static int PORT = 9200;
+    private static String SCHEME = "http";
+    private static String METHOD = "POST";
+    private static String END_POINT = "/anomalies/doc";
+    private Object object;
+
+    public ElasticSearchService(Object Object) {
+        this.object = object;
```

Comment:
This looks like a bug -- the param name is `Object`, so it looks like you are assigning the member to itself.